var/spack/repos/builtin/packages/py-natsort/package.py

```diff
@@ -37,3 +37,7 @@ class PyNatsort(PythonPackage):
     version("4.0.1", sha256="1c1d29150938ca71f0943363a06765dbb2cea01f9c4d760ba880cc65f39baba0")
 
     depends_on("py-setuptools", type=("build"))
+
+    variant("icu", default=False, description="Use icu for locale sorting")
```

Comment:
I think technically:
```suggestion
    variant("icu", default=False, when="@5.0.3:", description="Use icu for locale sorting")
```
Ref: https://github.com/SethMMorton/natsort/commit/0b304af16822c857e20cbcbc9a4cc79d28816dcd

Reply:
agreed